compiler/rustc_data_structures/src/stable_hasher.rs

```diff
@@ -57,6 +57,9 @@ impl StableHasher {
     }
 }
 
+// SipHasher128 currently handles ensuring platform-independent results with
```

Comment:
Can you add unit tests that test this property of `StableHasher` itself? We want to make sure this property holds even if we eventually move `StableHasher` away from `SipHasher`.

Reply:
Sure, I'll add some tests.

Reply:
Tests added.